### PR TITLE
WRO-784: Updated Sandstone and React dependencies for 'my-theme-app'

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

Updated Sandstone and React dependencies to latest versions

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: my-theme-app/package.json [hunk 1]
```diff
@@ -33,15 +33,15 @@
     "dist/*"
   ],
   "dependencies": {
-    "@enact/core": "^3.0.0",
-    "@enact/i18n": "^3.0.0",
+    "@enact/core": "^4.5.0-alpha.1",
```
Comment (MikyungKim) [inline my-theme-app/package.json:36]: Please update the versions to rc.1.

File: my-theme-app/src/index.js [hunk 2]
```diff
@@ -1,11 +1,16 @@
 import 'web-animations-js';
-import {render} from 'react-dom';
+import {createRoot} from 'react-dom/client';
```
Comment (MikyungKim) [inline my-theme-app/src/index.js:2]: Would you please update this pattern to the same one in https://github.com/enactjs/cli/blob/master/docs/isomorphic-support.md?

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**